geonetwork/transform-mcp-iso19115-3/schema/iso19115-3/process/transform-19115-3-testing-urls.xsl

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:gco="http://standards.iso.org/iso/19115/-3/gco/1.0"
+                version="2.0"
+                exclude-result-prefixes="#all">
+
+  <!-- url substitutions to be performed -->
+  <xsl:variable name="urlSubstitutions">
+    <substitution match="geoserver.123.aodn.org.au" replaceWith="geoserver-gn3-integration.dev.aodn.org.au"/>
+    <substitution match="https?://thredds.aodn.org.au(:443)?" replaceWith="http://thredds-gn3-integration.dev.aodn.org.au"/>
+    <substitution match="catalogue-imos.aodn.org.au" replaceWith="catalogue-imos.dev.aodn.org.au"/>
+    <substitution match="https?://tilecache.aodn.org.au(:443)?" replaceWith="https://tilecache-gn3-integration.dev.aodn.org.au"/>
```

Comment:
I think all the stack endpoints are plain http (no load balancers - they cost money)

Reply:
Updated tilecache endpoint.  aws-wps endpoint is https so its OK.